src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -276,7 +276,7 @@ Status CoreWorkerPlasmaStoreProvider::Get(
     }
     RAY_RETURN_NOT_OK(
         FetchAndGetFromPlasmaStore(remaining, batch_ids, /*timeout_ms=*/0,
-                                   /*fetch_only=*/true, ctx.CurrentTaskIsDirectCall(),
+                                   /*fetch_only=*/false, ctx.CurrentTaskIsDirectCall(),
```

Comment:
@stephanie-wang did you mean this change specifically? This eliminates the extra wait requests generated by ray.get().

Reply:
Oh I meant to call `AsyncResolveObjects` in the raylet when fetch_only is true, but to not mark the worker as blocked. I've lost track of this codepath, maybe either change is fine?